src/commands/watch/filters.rs

```diff
@@ -185,3 +181,23 @@ fn standard_ignores() -> Vec<(String, Option<PathBuf>)> {
     .map(|pat| (pat.to_owned(), None))
     .collect()
 }
+
+#[derive(Debug)]
+struct CompositeFilterer {
+    filterers: Vec<Box<dyn watchexec::filter::Filterer>>,
```

Comment:
Nit: Perhaps put a `Vec<Box<dyn watchexec::filter::Filterer>>` behind a type alias? Not sure if that is cleaner so I'll leave that up to you.

Reply:
Hmm, to my mind it doesn't occur enough, or have a distinct semantic concept other than "vector of filterers", to merit the indirection.  I agree Rust's insistence on box-dyn ceremony makes it harder to read than it needs to be though.  If we want to do this I think I'd alias the box-dyn and make this `Vec<AnyFilterer>`...